demosplan/DemosPlanCoreBundle/Controller/Document/DemosPlanDocumentController.php

```diff
@@ -634,103 +634,101 @@ public function elementAdminListAction(
-        }
-
-        // bereinige die Dateien nach einem Export oder einem Abbruch auf der Zwischenseite
+        /*
+         * Remove files from the session when import was cancelled
+         * @see DemosPlanDocumentController::importElementDirToArraySaveHashInSession
+         * */
         if ($session->has('element_import_list')) {
             $this->cleanElementImport($request, $currentProcedureArray['id'], $currentUser->getUser());
         }
 
-        // Template Variable aus Storage Ergebnis erstellen(Output)
-        // die Rekursion der Elemente wird im Twig erledigt, hole nur top-level elements (Elements ohne parent) aus dem repository,
-        // jedoch ohne solche die eines der Kriterien aus $filterCriteria erfüllen, diese sollen momentan nicht im template angezeigt werden
-        $result['elementlist'] = $elementsService->getTopElementsByProcedureId(
-            $procedure,
-            $filterCriteria,
-            true
-        );
-
-        $templateVars['list'] = $result;
-
-        $templateVars['procedure'] = $procedureHandler->getProcedure($procedure);
-
+        /**
+         * Collect errors, if any, produced during the saving of imported elements
+         * and store them in a variable to display in the UI.
+         *
+         * @see DemosPlanDocumentController::saveImportedElementsAdminAction
+         * */
         $errorReports = $session->getFlashBag()->get('errorReports');
         $templateVars['errorReport'] = [];
 
         if ((is_countable($errorReports) ? count($errorReports) : 0) > 0) {
             $templateVars['errorReport'] = $errorReports[0];
         }
 
-        $title = 'elements.dashboard';
+        $templateVars['procedure'] = $procedureHandler->getProcedure($procedure);
 
-        // Füge die kontextuelle Hilfe dazu
         $templateVars['contextualHelpBreadcrumb'] = $breadcrumb->getContextualHelp($title);
-        // @improve T14122
         $mapOptions = $mapService->getMapOptions($procedure);
         $templateVars['procedureDefaultInitialExtent'] = $mapOptions->getProcedureDefaultInitialExtent();
 
         $procedureSettings = $currentProcedureArray['settings'];
 
-        // This redirect ensures that any messagesBag notifications created in events related to this action are
-        // properly transformed into FlashBag messages, since the method for that is called in the
-        // DemosPlanResponseListener, see bug T17790.
-        if (0 !== (is_countable($requestPost) ? count($requestPost) : 0)) {
-            return $this->redirectToRoute('DemosPlan_element_administration', ['procedure' => $procedure]);
-        }
-
         return $this->renderTemplate(
             '@DemosPlanCore/DemosPlanDocument/elements_admin_list.html.twig',
-            compact('templateVars', 'procedure', 'title', 'procedureSettings')
+            compact('templateVars', 'title', 'procedureSettings')
```

Comment:
Am I right, that the `'/verfahren/{procedure}/verwalten/planunterlagen` elementAdminListAction-Controller now would look more or less the same as the `/verfahren/{procedure}/verwalten/karte` elementAdminListAction-Controller and one of them can be deleted in the End?

Reply:
hey @salisdemos exactly, one of them can be deleted, I would recommend to delete the /verwalten/karten